keylime/tpm/ec_crypto_helper.py

```diff
@@ -0,0 +1,127 @@
+# pylint: disable=protected-access
+
+from typing import Any, Optional
+
+from cryptography.hazmat import backends
+from cryptography.hazmat.backends.openssl.backend import Backend as OpenSSLBackend
+from cryptography.hazmat.bindings.openssl import binding
+from cryptography.hazmat.primitives.asymmetric.ec import EllipticCurvePrivateKey, EllipticCurvePublicKey
+
+
+class EcCryptoHelper:
+    instance: Optional["EcCryptoHelper"] = None
+
+    @staticmethod
+    def get_instance() -> "EcCryptoHelper":
+        """Create and return a singleton EcCryptoHelper"""
+        if not EcCryptoHelper.instance:
+            EcCryptoHelper.instance = EcCryptoHelper()
+        return EcCryptoHelper.instance
+
+    def __init__(self) -> None:
+        self.backend = backends.default_backend()
```

Comment:
Why this is needed? Can be because some side effects, like some object creation, loading of the *.so ... ? If so, can you add a comment? Maybe should not be stored as a field in the class?

Reply:
It's not need anymore... I removed it.